.github/workflows/publish.js.yml

```diff
@@ -13,12 +13,26 @@ jobs:
     runs-on: macos-12
 
     env:
-      XCODE_VERSION: 14.1
+      XCODE_VERSION: 14.2
+
+      # Real devices
       ZIP_PKG_NAME_IOS: "WebDriverAgentRunner-Runner.zip"
       PKG_PATH_IOS: "appium_wda_ios"
       ZIP_PKG_NAME_TVOS: "WebDriverAgentRunner_tvOS-Runner.zip"
       PKG_PATH_TVOS: "appium_wda_tvos"
 
+      # Simulator - x86_64
+      ZIP_PKG_NAME_IOS_SIM_X86_64: "WebDriverAgentRunner-Build-Sim-x86-64.zip"
```

Comment:
I see that prefixes are the same everywhere. Would it make sense to only have prefixes defined and generate suffixes in the runtime script based on the current platform value?